.github/workflows/ci.yml

```diff
@@ -93,7 +93,7 @@ jobs:
           lake --quiet exe generate-manual --depth 2 --verbose --draft --without-html-single --output "_out/draft"
 
       - name: Upload static files as artifact
-        uses: actions/upload-pages-artifact@v3
+        uses: actions/upload-pages-artifact@7b1f4a764d45c48632c6b24a0339c27f5614fb0b # v4.0.0
```

Comment:
Do we have any hidden files in `_out` that we need to deploy? If so then we may need to stick with v3 here.